packages/account/src/Sections/Verification/ProofOfOwnership/file-uploader.jsx

```diff
@@ -44,7 +44,7 @@ const FileUploader = ({
         await updateErrors(index, item_index, sub_index);
     };
     return (
-        <div className={`file-uploader ${class_name}`}>
+        <div className={`poo-file-uploader ${class_name}`}>
```

Comment:
nit: May be classNames() can be used rather than template string

Reply:
Alright, sure will refactor) but it is not related to my PR :)